File: Java/p11479.java
import java.util.Scanner;

public class p11479 {
    public static void main (String [] args) {
        Scanner input = new Scanner(System.in);
        int kasus = input.nextInt();
        for (int i = 0; i < kasus; i++) {
            long num1 = input.nextLong();
            long num2 = input.nextLong();
            long num3 = input.nextLong();

            if (num1 == num2 && num2 == num3 && num3 == num1) {
                System.out.printf("Case %d: Equilateral\n", i+1);
            } else if (num1+num2 > num3 && num1+num3 > num2 && num2+num3 > num1) {
                if (num1 == num2 || num1 == num3 || num2 == num3) {
                    System.out.printf("Case %d: Isosceles\n", i+1);
                } else {
                    System.out.printf("Case %d: Scalene\n", i+1);
                }
            } else {
                System.out.printf("Case %d: Invalid\n", i+1);
            }
        }
    }
}

File: Python/p11479.py
kasus = int(input())
temp = 0
while temp != kasus:
    masukan = input()
    masukan2 = masukan.split()
    num1 = int(masukan2[0])
    num2 = int(masukan2[1])
    num3 = int(masukan2[2])

    if num1+num2 <= num3 or num2+num3 <= num1 or num3+ num1 <= num2:
        print("Case {:d}: Invalid".format(temp+1))
    elif num1 == num2 == num3 :
        print("Case {:d}: Equilateral".format(temp+1))
    elif num1 == num2 or num2 == num3 or num3 == num1:
        print("Case {:d}: Isosceles".format(temp+1))
    else:
        print("Case {:d}: Scalene".format(temp+1))

    temp += 1